Examples/PlatformExample.cs

```diff
@@ -66,15 +66,15 @@ public static async Task Main(string[] args)
             // Create new client with specified authorization token
             //
 
-            influxDB = InfluxDBClientFactory.Create("http://localhost:9999", authorization.Token);
+            influxDB = new InfluxDBClient("http://localhost:9999", authorization.Token);
```

Comment:
Please refactor to `using(...)`:

```suggestion
            using (var influxDB = new InfluxDBClient("http://localhost:9999",
                       "my-user", "my-password"))
            {
```